### ISSUE TITLE
early stopping

### ISSUE THREAD

**Issue by innnfinity** [LunaBlack/KGAT-pytorch#29]
Thank you for the data and code!

But the early stopping here is done on the test set, shouldn't you use the validation set?

Looking forward to your reply very much.
